source/user-manual/reference/ossec-conf/wodle-syscollector.rst

```diff
@@ -165,6 +165,8 @@ Enables the scan of the processes.
 | **Allowed values** | yes, no |
 +--------------------+---------+
 
+.. _syscollector_hotfixes:
```

Comment:
This label is duplicated. It's also included in `/source/user-manual/capabilities/system-inventory/available-inventory-fields.rst`